### DISCUSSION TITLE
can not Install by pip, conda, or mamba

### DISCUSSION THREAD

**Discussion by jianwei-github** [torchmd/torchmd-net#342] [Q&A]
File "/tmp/pip-req-build-upx27ul9/setup.py", line 22, in <module>
          use_cuda = torch.cuda._is_compiled() if not force_cpu_only else False
module 'torch.cuda' has no attribute '_is_compiled'

**Comment by stefdoerr** [comment]
Hi, can you report what the `nvidia-smi` command gives you?

**Comment by jianwei-github** [comment]
I installed the cuda113 in&nbsp;conda enviroment.&nbsp;


this is command "conda install conda-forge::torchmd-net" results:


Collecting package metadata (repodata.json): \ WARNING conda.models.version:get_matcher(556): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.6.0.*, but conda is ignoring the .* and treating it as 1.6.0
WARNING conda.models.version:get_matcher(556): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.9.0.*, but conda is ignoring the .* and treating it as 1.9.0
WARNING conda.models.version:get_matcher(556): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.8.0.*, but conda is ignoring the .* and treating it as 1.8.0

**Comment by stefdoerr** [comment]
Can you please run `nvidia-smi` in a command line and show me the result?
Also run `conda list` so that I know what packages are installed in that env

**Comment by jianwei-github** [comment]
After downloading the pakage, I used "python setup.py install ". But still had errors.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "torchmd/torchmd-net"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月25日(星期三) 下午4:50
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [torchmd/torchmd-net] can not Install by pip, conda, or mamba (Discussion #342)






Hi, can you report what the nvidia-smi command gives you?

**Comment by stefdoerr** [comment]
If you don't answer my questions I cannot help you, sorry

**Comment by jianwei-github** [comment]
I'm sorry. I was just&nbsp; for dinner, and not saw your email.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "torchmd/torchmd-net"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月25日(星期三) 下午5:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [torchmd/torchmd-net] can not Install by pip, conda, or mamba (Discussion #342)






If you don't answer my questions I cannot help you, sorry

**Comment by jianwei-github** [comment]
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "torchmd/torchmd-net"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月25日(星期三) 下午5:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [torchmd/torchmd-net] can not Install by pip, conda, or mamba (Discussion #342)






Can you please run nvidia-smi in a command line and show me the result?
Also run conda list so that I know what packages are installed in that env
